### DISCUSSION TITLE
Auto Crop Borders (black bars) Function

### DISCUSSION THREAD

**Discussion by kaelsonofkrypto** [nagadomi/nunif#95] [General]
I completely forgot to add this as a request, sometimes 2d to 3d conversion will be more accurate if black bars are cropped out on movies that have black bars. This would be a good addition to add if possible. But if its difficult to add don't worry about it. 

Thanks in advance for all your help.

**Comment by nagadomi** [comment]
You can specify ffmpeg's video filters with `-vf (src)` option. Crop is implemented in `crop` filter.
For example, crop 10 pixels from the left and right borders, and 20 pixels from the top and bottom borders
`crop=in_w-2*10:in_h-2*20`
For more details and examples, see `crop` section in https://ffmpeg.org/ffmpeg-filters.html

`cropdetect` filter can auto-detect crop size from video. However, it seems that the results cannot be passed to `crop`.
I am not very familiar with ffmpeg's video filter.
